src/olympia/promoted/views.py

```diff
@@ -0,0 +1,35 @@
+import stripe
+
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+
+import olympia.core.logger
+
+from olympia.amo.decorators import post_required
+
+from .utils import create_stripe_webhook_event
+
+
+log = olympia.core.logger.getLogger("z.promoted")
+
+
+@post_required
```

Comment:
This is written as a standard django view, but we want a Rest Framework api view because its in the api. 

`@api_view(['POST'])`

Reply:
ha ok